tester/src/test/java/com/silanis/esl/sdk/examples/PackageReferencedConditionsExampleTest.java

```diff
@@ -0,0 +1,40 @@
+package com.silanis.esl.sdk.examples;
+
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasSize;
+
+import com.silanis.esl.sdk.ReferencedConditions;
+import com.silanis.esl.sdk.ReferencedDocument;
+import com.silanis.esl.sdk.ReferencedField;
+import com.silanis.esl.sdk.ReferencedFieldConditions;
+import java.util.List;
+import org.junit.Test;
+
+public class PackageReferencedConditionsExampleTest {
+
+    @Test
+    public void verifyResult() {
+        PackageReferencedConditionsExample example = new PackageReferencedConditionsExample();
+        example.run();
+
+        ReferencedConditions packageLevelRefConditions = example.getPackageLevelRefConditions();
+        ReferencedConditions documentLevelRefConditions = example.getDocumentLevelRefConditions();
+        ReferencedConditions fieldLevelRefConditions = example.getFieldLevelRefConditions();
```

Comment:
As with the other examples, do not use the private method but directly access the public field.

Reply:
Updated